app/knights/basic_stats.py

```diff
@@ -0,0 +1,28 @@
+class Knight:
+    def __init__(self,
+                 name: str,
+                 power: int,
+                 hp: int,
+                 armour: list,
+                 weapon: dict,
+                 potion: dict
+                 ) -> None:
```

Comment:
fix indentation

Reply:
Can you help me with indicating how to change it? Flake8 and IDE say, that it's okay. Thank you

Reply:
I mentioned an example of good indentation above